pkg/config/settings.go

```diff
@@ -44,22 +45,18 @@ const (
 
 var DefaultCr v1.IBMBlockCSI
```

Comment:
I didn't want to change it because a lot of places are using it in the code, so I didn't want that it will be part of this change, I can open a ticket if you think it ok

Reply:
I think the best approach is to merge said refactor before merging this PR, as a preparation.
that way, the `develop` branch won't contain this ambiguity at any point in time

Reply:
[257](https://github.com/IBM/ibm-block-csi-operator/pull/257) PR solves it

Reply:
I pushed the changes to develop and to this branch